lib/config.js

```diff
@@ -46,12 +46,13 @@ module.exports = function (options) {
     if (!url.parse(options.dbUrl).auth) {
       log.warn('config', 'Authentication details missing from database URL')
     }
+  } else {
+    log.warn('config', 'No database URL provided, falling back to PouchDB')
   }
 
   if (options.inMemory) {
     config.db.db = require('memdown')
   }
 
-  require('deep-freeze')(config)
```

Comment:
I introduced this mostly for debugging, so I could find all spots where it was initially modified all over the place, also inside the plugins api stuff.
Most of this isn't in hoodie-server anymore and now I deleted it because I needed to modify the config object with the server's secret.
As everything is now encapsulated in components it might no longer be necessary, but we can add it again later if necessary.